src/v2i-hub/FLIRCameraDriverPlugin/src/FLIRPedestrianPresenceTrackingProcessor.cpp

```diff
@@ -258,4 +264,15 @@ namespace FLIRCameraDriverPlugin
         FILE_LOG(tmx::utils::LogLevel::logWARNING) << "Ped presence data subscription status: " << subscrStatus;
         return false;
     }
+
+    double roundNearZeroDoubles(double value) {
+        // Preprocess doubles to avoid scientific notation in JSON serailization
+        if (value < 0.001 && value > -0.001) {
```

Comment:
```suggestion
        if (std::abs(value) < 0.001) {
```
If we want to keep the bound symmetric around 0 this may be cleaner